tools/device_configuration_checker.py

```diff
@@ -299,35 +300,38 @@ def _run_tests_on_know_good_image(args, device):
 
     image = None
     if device.model.lower() == "beagleboneblack":
-        image = image_directory_path
+        image = "good-image.tar.bz2"
     elif device.model.lower() == "edison":
-        image = os.path.join(
-            image_directory_path,
-            "ostro-image-edison.ext4")
+       image = "good-image.ext4"
```

Comment:
In general, machine-specific dependencies are kind of annoying as adding\modifying devices means several places in AFT codebase needs to be updated. (qatestcase.py is another one with machine specific dependencies).

Maybe a external config file should be created for this, but it's a different task.